### ISSUE TITLE
Deprecated use of async_forward_entry_setup?

### ISSUE THREAD

**Issue by anthonyprescott** [caronc/ha-ultrasync#43]
Hi guys,

This is probably old news, so please close if required.  The latest version of HA core, throw this:

Log details (WARNING)
Logger: homeassistant.helpers.frame
Source: helpers/frame.py:188
First occurred: 4:22:59 PM (1 occurrences)
Last logged: 4:22:59 PM

Detected that custom integration 'ultrasync' calls async_forward_entry_setup for integration, ultrasync with title: 192.168.0.20 and entry_id: xxxxxxxx, which is deprecated and will stop working in Home Assistant 2025.6, await async_forward_entry_setups instead at custom_components/ultrasync/__init__.py, line 68: entry.async_create_task(, please create a bug report at https://github.com/caronc/ha-ultrasync/issues

**Comment by Lachyc88** [comment]
Yeah, I've just started using HASS and got it working but noticed these errors.
Hopefully further modifications/updates can be made to keep this integration working.

**Comment by MoFtZ** [comment]
This should [already be fixed](https://github.com/caronc/ha-ultrasync/pull/40). But needs a new version to be published.

**Comment by Lachyc88** [comment]
Perfect, thanks for letting us know.
